### DISCUSSION TITLE
Show timeout max value warning in documentation

### DISCUSSION THREAD

**Discussion by dhimitrios12** [SupremeTechnopriest/react-idle-timer#316] [Ideas]
I think it would be good to have mentioned somewhere in the documentation (maybe a warning)that there is a max value the <code>timeout </code> property can take. 
In my project we set the timeout in minutes and the QA was testing for some big values. When we were converting it to milliseconds the value was surpassing the max value that <code>setTimeout(...)</code> can handle and the prompt was shown immediately as soon as the user open the app. 

I know it is not good to have big values for idle time for security reasons. It was just a testing scenarion.

Some docs: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#:~:text=Maximum%20delay%20value,the%20timeout%20being%20executed%20immediately.

**Comment by SupremeTechnopriest** [comment]
I will make the hook throw if you set a value that is too large.  I will also make a note in the docs.

**Reply by SupremeTechnopriest** [reply]
I have added a sanity check that will throw an error if you supply a timeout value that exceeds a 32 bit signed integer.  It will be released with 5.5.0 and is in the current release candidate. `npm i react-idle-timer@next`

**Reply by dhimitrios12** [reply]
That is great. Thanks.

**Reply by SupremeTechnopriest** [reply]
No problem!  I will also make a note in the docs when I update them later this week.
